src/main/scala/com/evolutiongaming/cluster/pubsub/OptimiseSubscribe.scala

```diff
@@ -32,15 +32,15 @@ object OptimiseSubscribe {
   type Listener[F[_]] = OnMsg[F, Any]
 
 
-  def of[F[_] : Concurrent : Parallel]: F[OptimiseSubscribe[F]] = {
+  def of[F[_] : Async: Parallel]: F[OptimiseSubscribe[F]] = {
     for {
       serialMap <- SerialMap.of[F, String, Subscription[F]]
     } yield {
       apply(serialMap)
     }
   }
 
-  def apply[F[_] : Sync : Parallel](serialMap: SerialMap[F, String, Subscription[F]]): OptimiseSubscribe[F] = {
+  def apply[F[_] : Sync: Parallel](serialMap: SerialMap[F, String, Subscription[F]]): OptimiseSubscribe[F] = {
```

Comment:
`Sync` can be replaced with `MonadCancelThrow`

Reply:
addressed in [3025814](https://github.com/evolution-gaming/pubsub/pull/163/commits/302581413fe6a1632a1ac021e0731f31ec1add31)